app/helpers/application_helper.rb

```diff
@@ -148,10 +160,13 @@ def ssn(number)
     end
   end
 
-  def masked_ssn(number)
+  def masked_ssn(number, include_content_tag: true)
     # pad with leading 0s if we don't have enough characters
     number = number.to_s.rjust(9, '0') if number.present?
-    content_tag :span, number.to_s.gsub(HudUtility2024::SSN_RGX, 'XXX-XX-\3')
+    value = number.to_s.gsub(HudUtility2024::SSN_RGX, 'XXX-XX-\3')
+    return value unless include_content_tag
+
+    ActionController::Base.helpers.content_tag :span, number.to_s.gsub(HudUtility2024::SSN_RGX, 'XXX-XX-\3')
```

Comment:
Since this is in `ApplicationHelper`, I don't think you need:
```suggestion
    content_tag :span, number.to_s.gsub(HudUtility2024::SSN_RGX, 'XXX-XX-\3')
```

Reply:
this has been updated [here](https://github.com/greenriver/hmis-warehouse/pull/5183/commits/3a52aa0755560c964a26f94e516edfd1904b94a3)